library/java/src/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -34,9 +34,9 @@ String resolveTemplate(String templateYAML) {
     // TODO(@buildbreaker): Implement actual domain template resolution
     String resolvedConfiguration =
         templateYAML.replace("{{ domain }}", "s3.amazonaws.com")
-            .replace("{{ connect_timeout }}", String.format("%ss", connectTimeoutSeconds))
-            .replace("{{ dns_refresh_rate }}", String.format("%ss", dnsRefreshSeconds))
-            .replace("{{ stats_flush_interval }}", String.format("%ss", statsFlushSeconds));
+            .replace("{{ connect_timeout }}", String.format("%s", connectTimeoutSeconds))
```

Comment:
In this PR I am moving all the added `s`s to the template so that the platform only has to inject the number. The absence of `s` in the seconds value for iOS was causing crashes.

Reply:
The change seems reasonable. What was the crash you were seeing?

Reply:
Because these fields are duration fields they needed the trailing `s` to be interpreted as seconds. The iOS templating removed that so envoy was failing to start because `30` (only the number) is an invalid value for the duration field. To fix I added the trailing `s` to the config itself, and deleted here to create consistency. Also this way we have the same behavior in both platforms (i.e both refresh rates are seconds, rather than letting the platform change the duration unit independently).

I am going to update here to make it explicit in the variable name that this value is in seconds.

Reply:
Ah, I see. Yes that makes sense

Reply:
https://github.com/lyft/envoy-mobile/pull/448 for reference